cypress/integration/004-trap-registration-number.js

```diff
@@ -21,12 +21,59 @@ describe('Trap Registration Number page ', function () {
 
   it('should allow access if the user visits all the pages in order', function () {
     cy.visit('/trap-registration-number');
-    cy.get('h1').should('contain', 'Trap Registration Number');
+    cy.get('h1').should('contain', 'What is your trap registration number?');
   });
 
-  it('main button should navigate to postcode', function () {
+  it('Valid input followed by continue button should navigate to postcode', function () {
     cy.visit('/trap-registration-number');
+    cy.get('input').type('12345');
     cy.get('#main-content form button.naturescot-forward-button').click();
     cy.url().should('include', '/postcode');
   });
+
+  it('Valid input followed by continue button should navigate to postcode', function () {
```

Comment:
This should probably have a more descriptive title - right now it's the same as the previous test.

```diff
---  it('Valid input followed by continue button should navigate to postcode', function () {
+++  it('Valid, but whitespace padded, input followed by continue button should navigate to postcode', function () {
```

Reply:
Resolved in 318000903de1d74a6efe69c16c1004e4acdf7f3a.